MApp.js

```diff
@@ -0,0 +1,110 @@
+document.addEventListener('DOMContentLoaded', () => {
```

Comment:
The main functionality of the app which is to fetch movie data from Ghibli API then displaying them are running properly but comment function isn't complete. Also CSS could use more work since HTML elements are out of place in many occasions. Please spend time practicing and reviewing CSS since it's an important skill to be proficient at. As we move toward backend database in unit 3, keep practicing, make your own projects in free time.